tasks/main.yml

```diff
@@ -1,9 +1,16 @@
 ---
-- name: Remove externally managed from Debian 12
-  ansible.builtin.import_tasks: debian12.yml
-  when:
-    - ansible_distribution == "Debian"
-    - ansible_distribution_major_version|int >= 12
+# Remove EXTERNALLY-MANAGED if pip_externally_managed is true and python3 version is 3.11 or higher.
+# Related issue: https://github.com/geerlingguy/ansible-role-pip/issues/57
+- name: Get installed python3 version
+  ansible.builtin.shell: python3 --version | awk '{split($2, a, "."); print a[1] "." a[2]}'
```

Comment:
Instead of running `python3 --version`, do you think that we should replacing it with "{{ ansible_facts.python.version.major }}.{{ ansible_facts.python.version.minor }}"?